pkg/client/src/app/pages/applications/manage-imports/manage-imports.tsx

```diff
@@ -351,9 +351,9 @@ export const ManageImports: React.FC = () => {
                     <KebabDropdown
                       dropdownItems={[
                         <DropdownItem
-                          key="download-csv-sample"
+                          key="download-csv-template"
                           component={
-                            <a href="/sample_application_import.csv" download>
```

Comment:
It looks it is needed fix a route or an existence of the sample CSV file. "hub/importsummaries/download" path downloads previously uploaded CSV Import file which could be different to the sample one